_layouts/default.html

```diff
@@ -164,7 +164,7 @@ <h1>{{ page.title }}</h1>
                         Current OMERO Version:&nbsp;&nbsp;&nbsp;5.4.0
```

Comment:
Where is the above `5.4.0` used?

Reply:
That's been replaced with 5.4.1 in a merged PR, I just haven't merged the branch into this one because they're not conflicting.

Reply:
PR merged for the 5.4.1 release https://github.com/openmicroscopy/ome-help/pull/277